pcs_test/tier0/lib/corosync/test_live.py

```diff
@@ -78,7 +76,7 @@ def setUp(self):
         self.mock_runner = mock.MagicMock(spec_set=CommandRunner)
 
     def path(self, name):
-        return os.path.join(settings.corosync_binaries, name)
+        return os.path.join(settings.corosync_execs, name)
```

Comment:
We want to avoid joining paths in tests as well, please remove this function. Tests should use the same settings variables as production code.